src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,60 @@
+package main.java.parser;
+
+import main.java.command.*;
```

Comment:
Perhaps you could look into whether this wildcard import is necessary. While I understand that you will need all commands, CS2103T coding standards seem to discourage the use of wildcard imports.